app/app.go

```diff
@@ -19,6 +19,10 @@ func Handle(repoList map[string]string, projectTypeToCheck string, filter string
 			output_channel <- output.Print(fmt.Sprintf("\nChecking '%s' (%s) repos...", projectType, path))
 
 			visit := func(visitedPath string, info os.FileInfo, err error) error {
+				if err != nil {
+					output_channel <- output.Error(err.Error())
```

Comment:
I've merged an update that allows an error to be output without exiting the whole application, as if you have a few folders in your `~/.cig.yaml` and only one of them doesn't exist you don't want this to stop the execution of the application.

With that in mind could you just tweak the output to:

```go
output_channel <- output.Error(fmt.Sprintf("- %s", err.Error()))
```

so it matches the output when a directory is being listed.